scss/_patterns_logo-section.scss

```diff
@@ -73,18 +77,18 @@
     .p-logo-section__item {
       display: inline-block;
       margin: 0;
-      margin-bottom: calc(-1 * $logo-section-offset-small); // pull the next row of logos up
+      margin-bottom: calc(-1 * $logo-section-offset-medium); // pull the next row of logos up
```

Comment:
We talked about making those changes to default `p-logo-section` and deprecating the `--dense` variant.